ml-agents/mlagents/trainers/tensorflow_to_barracuda.py

```diff
@@ -1533,10 +1533,15 @@ def convert(
         barracuda.print_known_operations(known_classes, known_activations)
 
     # Load Tensorflow model
-    print("Converting %s to %s" % (source_file, target_file))
-    f = open(source_file, "rb")
-    i_model = tf.GraphDef()
-    i_model.ParseFromString(f.read())
+    if isinstance(source_graph, str):
```

Comment:
Can you make this two separate parameters instead of overloading it? Or maybe have two entry points, one that takes a file name and one that takes a graph instance?

Reply:
Good call, fixed!